COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
+You can use a multiplatform project with native targets as a CocoaPods dependency (Kotlin Pod). You can include such a dependency
+  in a `Podfile` of an Xcode project  by its name and path to the project directory containing the generated `Podspec`. 
+This dependency will be automatically built (and rebuilt) along with this project. 
+Such approach simplifies importing to Xcode by removing a need to write corresponding Gradle tasks and Xcode build steps manually.
+ 
+You can add dependencies between:
+* [A Kotlin Pod and an Xcode project with one target](#add-a-dependency-between-a-kotlin-pod-and-xcode-project-with-one-target)
+* [A Kotlin Pod and an Xcode project with several targets](#add-a-dependency-between-a-kotlin-pod-with-an-xcode-project-with-several-targets)
+* [Several Kotlin Pods and an Xcode project](#add-dependencies-between-multiple-kotlin-pods-and-an-xcode-project)
+
+> To correctly import the dependencies into the Kotlin/Native module, the
+`Podfile` must contain either [`use_modular_headers!`](https://guides.cocoapods.org/syntax/podfile.html#use_modular_headers_bang)
+or [`use_frameworks!`](https://guides.cocoapods.org/syntax/podfile.html#use_frameworks_bang)
+directive.
+{:.note}
+
+### Add a dependency between a Kotlin Pod and Xcode project with one target
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`) 
+of your Kotlin project.  
+    This step helps synchronize your Xcode project with Kotlin Pod dependencies by calling `pod install` for your `Podfile`.
+3. Specify the minimum target version for the Pod library.
+    > If you don't specify the minimum target version and a dependency Pod requires a higher deployment target, you may get an error.
+    {:.note}
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+        
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            ios.deploymentTarget = "13.5"
+            pod("AFNetworking", "~> 4.0.0")
+            podfile = project.file("../ios-app/Podfile")
+        }
+    }
+    ```
+    
+    </div>
 
+4. Add the name and path of the Kotlin Pod you want to include in the Xcode project to `Podfile`.
+
+    <div class="sample" markdown="1" theme="idea" mode="ruby" data-highlight-only>
+    
     ```ruby
-    target 'my-ios-app' do
-        pod 'my_kotlin_library', :path => 'path/to/my-kotlin-library'
+    use_frameworks!
+    
+    platform :ios, '9.0'
+    
+    target 'ios-app' do
+            pod 'kotlin_library', :path => '../kotlin-library'
     end
     ```
+    
+    </div>
+
+5. Re-import the project.
+
+### Add a dependency between a Kotlin Pod with an Xcode project with several targets
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
```

Comment:
1. Create an Xcode project with a `Podfile` if you haven’t done so yet.

Reply:
✅